scalajslib/test/src/mill/scalajslib/HelloJSWorldTests.scala

```diff
@@ -28,8 +28,8 @@ object HelloJSWorldTests extends TestSuite {
   }
 
   object HelloJSWorld extends TestUtil.BaseModule {
-    val scalaVersions = Seq("2.13.3", "3.0.0-RC1", "2.12.12", "2.11.12")
-    val scalaJSVersions = Seq("1.8.0", "1.3.1", "1.0.1")
```

Comment:
Are these versions no longer supported? AFAIK, those partly changed/extended their API, and those tests make sure we can still build against them.

Reply:
I thought we could kill them, but I'll revert this and leave it for another PR